examples/simulation/shower-event.py

```diff
@@ -2,72 +2,89 @@
 import astropy.units as u
 from grand import ECEF, LTP
 from grand.simulation import Antenna, ShowerEvent, TabulatedAntennaModel
-#from grand.simulation import ShowerEvent, TabulatedAntennaModel
-#from grand.simulation.antenna .generic import compute_voltage
-import numpy as np
-from matplotlib import pyplot as plt
-from astropy.coordinates import BaseRepresentation, CartesianRepresentation
+
+import os
+grand_astropy = True
+try:
+    if os.environ['GRAND_ASTROPY']=="0":
+        grand_astropy=False
+except:
+    pass
+    
+# Note: This examples requires some test data to be downloaded localy. You can
+# get these data from the grand store as:
+#
+# wget https://github.com/grand-mother/store/releases/download/101/HorizonAntenna_EWarm_leff_loaded.npy
+#
+# mkdir -p tests/simulation/data/zhaires
+# cd tests/simulation/data/zhaires
+# wget https://github.com/grand-mother/store/releases/download/101/zhaires-test.tar.gz
+# tar -xzf zhaires-test.tar.gz
+# cd -
+
 
 # Load the radio shower simulation data
-showerdir = '../../tests/simulation/data/zhaires'
-shower = ShowerEvent.load(showerdir)
+shower = ShowerEvent.load('tests/simulation/data/zhaires')
 if shower.frame is None:
     shower.localize(39.5 * u.deg, 90.5 * u.deg) # Coreas showers have no
                                                 # localization info. This must
                                                 # be set manually
 
-print("Shower frame=",shower.frame)
-print("Zenith (Zhaires?!) =",shower.zenith)
-print("Azimuth (Zhaires?!) =",shower.azimuth)
-print("Xmax=",shower.maximum)
-#shower.core=CartesianRepresentation(0, 0, 2900, unit='m')
-print("Core=",shower.core)
-
-
 # Define an antenna model
 #
 # A tabulated model of the Butterfly antenna is used. Note that a single EW
 # arm is assumed here for the sake of simplicity
 
-antenna_model = TabulatedAntennaModel.load('./HorizonAntenna_EWarm_leff_loaded.npy')
+antenna_model = TabulatedAntennaModel.load(
+    'HorizonAntenna_EWarm_leff_loaded.npy')
 
 # Loop over electric fields and compute the corresponding voltages
 for antenna_index, field in shower.fields.items():
     # Compute the antenna local frame
     #
     # The antenna is placed within the shower frame. It is oriented along the
-    # local magnetic North by using an ENU/LTP frame (x: East, y: North, z: Upward)
-    antenna_location = shower.frame.realize_frame(field.electric.r)
-    print(antenna_index,"Antenna pos=",antenna_location)
+    # local magnetic North by using an ENU/LTP frame (x: East, y: North, z:
+    # Upward)
+    
+    if grand_astropy:
+        print("field", field.electric.r, shower.frame)
 
-    antenna_frame = LTP(location=antenna_location, orientation='NWU',magnetic=True, obstime=shower.frame.obstime)
-    antenna = Antenna(model=antenna_model, frame=antenna_frame)
+        # LWP: joins field.electric.r and shower.frame
+        antenna_location = shower.frame.realize_frame(field.electric.r)
```

Comment:
This wraps the Cartesian coordinates of the E field position, i.e. the antenna one, with an `astropy.CoordinatesFrame` allowing for frame transforms.